tests/conftest.py

```diff
@@ -70,3 +73,361 @@ def _sample_all_wsis(_sample_ndpi, _sample_svs, _sample_jp2, tmpdir_factory):
+        .join("sample_patch_extraction_jp2.csv")
+
+    if not pathlib.Path(jp2_csv_file_path).is_file():
+        print("\nDownloading sample patch extraction jp2 csv file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction_jp2.csv")
+        with open(jp2_csv_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction jp2 csv file")
+
+    return jp2_csv_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_jp2_read(tmpdir_factory):
+    """Sample pytest fixture for sample patch extraction jp2 read npy.
+    Download sample patch extraction jp2 read npy for pytest. """
+
+    jp2_read_file_path = tmpdir_factory.mktemp("data")\
+        .join("sample_patch_extraction_jp2read.npy")
+
+    if not pathlib.Path(jp2_read_file_path).is_file():
+        print("\nDownloading sample patch extraction jp2 read npy file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction_jp2read.npy")
+        with open(jp2_read_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction jp2 read npy file")
+
+    return jp2_read_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_read(tmpdir_factory):
+    """Sample pytest fixture for sample patch extraction read npy.
+    Download sample patch extraction read npy for pytest. """
+
+    read_file_path = tmpdir_factory.mktemp("data")\
+        .join("sample_patch_extraction_read.npy")
+
+    if not pathlib.Path(read_file_path).is_file():
+        print("\nDownloading sample patch extraction read npy file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction_read.npy")
+        with open(read_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction read npy file")
+
+    return read_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_svs(tmpdir_factory):
+    """Sample pytest fixture for sample patch extraction svs csv.
+    Download sample patch extraction svs csv for pytest. """
+
+    svs_file_path = tmpdir_factory.mktemp("data")\
+        .join("sample_patch_extraction_svs.csv")
+
+    if not pathlib.Path(svs_file_path).is_file():
+        print("\nDownloading sample patch extraction svs csv file")
+        r = requests.get("https://tiatoolbox.dcs.warwick.ac.uk/testdata/"
+                         "patchextraction/sample_patch_extraction_svs.csv")
+        with open(svs_file_path, "wb") as f:
+            f.write(r.content)
+    else:
+        print("\nSkipping sample patch extraction svs csv file")
+
+    return svs_file_path
+
+
+@pytest.fixture(scope="session")
+def _patch_extr_svs_header(tmpdir_factory):
```

Comment:
"""Sample pytest fixture for sample patch extraction svs_header csv.
    Download sample patch extraction svs_header csv for pytest.

"""

Reply:
Resolved in this commit https://github.com/TIA-Lab/tiatoolbox/pull/99/commits/f140ef4378c6618ae9d1f5cc7e47785c9c6da702